apps/funding-service/src/blockchain/index.ts

```diff
@@ -40,7 +42,7 @@ export const sendTransaction = async (params: {
 export const getProvider = async (chainId: number) => {
   if (!validChainIds.has(chainId)) throw new Error("Chain not supported");
   const provider = new ethers.providers.JsonRpcProvider(
-    validChainIds.get(chainId)
+    validChainIds.get(chainId)?.url
```

Comment:
@ItsLuized possibly not needed, run e2e without it and remove if it adds no value

Reply:
This is needed, tests without it throw: `TypeError: Invalid URL`